### DISCUSSION TITLE
Is there a unique identifier for a SABnzbd setup?

### DISCUSSION THREAD

**Discussion by shaiu** [sabnzbd/sabnzbd#2173] [Q&A] [answered]
I need a unique identifier for a SABnzbd setup to distinguish between multiple setups.
I'm working currently on integration for SABnzbd in HomeAssistant and it's valid to have multiple SABnzbd setups configured in it.
So a uuid for each setup could really help.

Thanks.

**Comment by thezoggy** [comment]
no we dont generate a uuid. why not just use the sab apikey?

**Reply by shaiu** [reply]
Because the user can change that so it's not unique per setup

**Comment by Safihre** [comment, marked as answer]
We don't have one, so far there hasn't been a use for it..
